routers/events/events.go

```diff
@@ -75,6 +81,32 @@ loop:
 		case <-shutdownCtx.Done():
 			go unregister()
 			break loop
+		case <-stopwatchTimer.C:
```

Comment:
This could be a little more efficient and happen in the modules/eventsource instead per uid.